src/notebook/notebookKernel.ts

```diff
@@ -78,6 +85,7 @@ export class JuliaKernel implements vscode.NotebookKernel {
 
     public async start() {
         if (!this._terminal) {
+            this._current_request_id = 0
```

Comment:
When restarting, or the like, this needs to be reset (at least in Jupyter, thats the behavior)